src/hdmf/data_utils.py

```diff
@@ -179,9 +179,15 @@ class GenericDataChunkIterator(AbstractDataChunkIterator):
             doc="Display a progress bar with iteration rate and estimated completion time.",
             default=False,
         ),
+        dict(
+            name="progress_bar_class",
+            type=Callable,
```

Comment:
For whatever reason, `typing.Callable` is a deprecated alias to `collections.abc.Callable`:
https://docs.python.org/3/library/typing.html#typing.Callable

Reply:
Probably because the more modern type is, simply, `callable`

Reply:
Just FYI this did not seem to work in practice when triggered with the GUIDE. Just opened https://github.com/catalystneuro/neuroconv/pull/861 so that all the pieces are in place to see the progress of a full conversion.

Reply:
Nevermind this is working fine, as shown on https://github.com/NeurodataWithoutBorders/nwb-guide/pull/778#issuecomment-2113243480